src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -65,6 +65,25 @@ function showHideStats() {
     }
 }
 
+// Toggles Rubric Questions excluding self
+function toggleExcludingSelfResultsForRubricStatistics(checkbox) {
+    if ($(checkbox).prop('checked')) {
+        $(checkbox).closest('.resultStatistics')
```

Comment:
I actually don't like the assumption that `feedbackQuestionRubricResultStatsTemplate.html` is always wrapped by `div` with class `.resultStatistics`. A more proper way could be adding class to the outer `div` in `feedbackQuestionRubricResultStatsTemplate.html` and using that classname as `.closet()`.

Reply:
Then it would make more sense to add a new class name for the outer div as `rubricResults`, because when instructor views results, the outer dive contains the resultStatistics as well as details about individual responses